src/app/main/main.component.ts

```diff
@@ -71,28 +72,56 @@ export class MainComponent implements OnInit, OnChanges {
     this.refreshCache();
   }
   async refreshDashBoard() {
+    let strandbarActivate: boolean;
+    if (moment().isBetween('2019-03-31', '2019-10-31', 'month')) {
```

Comment:
Set `strandbarActivate` directly to `(moment().isBetween('2019-03-31', '2019-10-31', 'month'))`